### ISSUE TITLE
Replace Deliveries table

### ISSUE THREAD

**Issue by stephenkao** [CDCgov/prime-reportstream#8958]

**Comment by chris-kuryak** [comment]
Awaiting confirmation from Platform if there will be any changes

**Comment by chris-kuryak** [comment]
Per Platform, there are no planned changes to this API for Universal Pipeline. We are unblocked to replace this table whenever we get the time.

**Comment by etanb** [comment]
@chris-kuryak was noticing some strange behavior in the table sorting in Daily Data with my new work...but it has to do with still utilizing this old table. I reverted back to pre-my-changes Daily Data and you can see in the video below, if you click the sort order icon a few times, it gets into a weird state, adding pagination and entries (there are only 4 entries, there should never be pagination). Unfortunately these issues persist, and just showcasing this as this might make removing the old tables an even higher priority.


[Private Zenhub Video](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBekxFQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b665297cfa57f36c376f550db23c3a36c5203635/oldTableIssues.mp4)

**Comment by chris-kuryak** [comment]
The strange behavior is not blocking functionality, right? 
But definitely heard on the poor user experience, and since this is a low-med lift, I have it already slotted as a low priority item for next sprint.

**Comment by etanb** [comment]
Not blocking, but certainly affecting it. It's not terrible, but something that should be addressed.

**Comment by chris-kuryak** [comment]
Carry over to next sprint

**Comment by chris-kuryak** [comment]
Moved to 5-points to accommodate the added ACs
